src/screens/login/Login.tsx

```diff
@@ -120,10 +49,12 @@ const Login = () => {
     };
   }, []);
 
-  const highlightUnderlayColor = colorWithOpacity(
-    isDark ? Colors.primary.dark : Colors.primary.light,
-    0.75,
-  );
+  useEffect(() => {
```

Comment:
errors can be handled in a better way, if you are showing only toast of the error, you can directly handle this part in the `useAppSelector` code only